tests/phpunit/tests/media.php

```diff
@@ -5290,6 +5290,51 @@ public function test_quality_with_image_conversion_file_sizes() {
 		}
 	}
 
+	/**
+	 * Test AVIF quality filters.
+	 *
+	 * @ticket 61614
+	 */
+	public function test_quality_with_avif_conversion_file_sizes() {
```

Comment:
Is this test relevant to Trac-53645? Might need you to explain to me what's changed here 😀

Reply:
no this test is for 6Trac-1614 and snuck in because I didn't clean up completely, I will remove it. Thanks!